CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 
 - Fix default entity insertion for a user
+- Fixed the SQL error on creating new injection model
```

Comment:
```suggestion
- Fixed `SQL` error when creating new injection model
```

Reply:
Sorry for my lack of knowledge on this process, I haven't used Github in years. Do I just hit "Commit suggestion", or do I need to make the change and re-push?